lib/beebee/storage/redis.ex

```diff
@@ -49,6 +49,25 @@ defmodule BeeBee.Storage.Redis do
     end
   end
 
+  @impl true
+  def delete(short_tag) do
+    with {:ok, ["OK", "OK"]} <-
+           Redix.pipeline(@process_name, [
+             ["DEL", url_key(short_tag)],
+             ["DEL", count_key(short_tag)]
+           ]) do
+      {:ok, short_tag}
```

Comment:
Dialyzer fails here because the Behavior expects `:ok` atom instead of an `{:ok, data}` tupe